Makefile

```diff
@@ -0,0 +1,26 @@
+APP_NAME=burrow
+
+QUAY_REPO?="unknown"
```

Comment:
Are you setting this in Circle env vars?

Reply:
Yes!